src/Latte/Loaders/FileLoader.php

```diff
@@ -27,7 +27,9 @@ public function getContent($file)
 			throw new \RuntimeException("Missing template file '$file'.");
 
 		} elseif ($this->isExpired($file, time())) {
-			touch($file);
+			if (@touch($file) === FALSE) {
+				trigger_error("File's modification time is in the future. Cannot update it: " . error_get_last()['message'], E_USER_WARNING);
```

Comment:
@dg I hope so. The touch() is called only when `isExpired()`, which in this case is compared to current `time()`.